packages/module/src/Message/Message.test.tsx

```diff
@@ -787,6 +789,13 @@ describe('Message', () => {
     render(<Message avatar="./img" role="user" name="User" content={IMAGE} />);
     expect(screen.getByRole('img', { name: /Multi-colored wavy lines on a black background/i })).toBeTruthy();
   });
+  it('should handle inline image correctly - they should not be rendered within p tags', () => {
```

Comment:
Couple of comments for this test:

- This is really more a nit for the naming of the test, but personally the test name should less be "Does something correctly" and more just what you have after the hyphen: `it("Does not render inline images with a <p> tag")`
- For the test itself, ti doesn't seem like we're really testing that a p tag isn't wrapped around the image? If the `pf-chatbot__message-and-actions` element is the wrapper of the image that is expected to not be a p tag, can we add an expectation that that element to be that tag?